src/containers/pages/PractitionerViews/PractitionerListView/index.tsx

```diff
@@ -118,7 +119,10 @@ const PractitionersListView = (props: PropsTypes) => {
 
   /** hook to load all practitioners and dispatch to them to store */
   useEffect(() => {
-    loadPractitioners(serviceClass, fetchPractitionersCreator);
+    loadPractitioners(serviceClass, fetchPractitionersCreator, signal);
+    return () => {
+      controller.abort();
+    };
```

Comment:
Might it be DRY'er to have a util function that does this instead of re-writing the anon function?
```
const controllerAborter = (controller) => { controller.abort() };
...
useEffect(() => {
  ...
  return controllerAborter(controller);
}, []);
```
(but with a better name than `controllerAborter`?)

Reply:
hmmm... maybe I could try and see if a custom hook will serve us better here.